app/main.py

```diff
@@ -88,6 +91,37 @@ def _get_client(self):
                                              self.client_id, self.client_secret)
         return KeyVaultClient(credentials)
 
+    def _get_tenant_id(self, tenant_id_from_config):
```

Comment:
Do you think we should be handling certain exceptions in this function. Like, can request.get() throw, should we just crash in this case?

Reply:
that's a good point. Let me add a try catch around and default the return value to id from config.